### ISSUE TITLE
0.9 does not seem to work

### ISSUE THREAD

**Issue by arnaudlimbourg** [sesh/piprot#40]
Hello,

When running the latest version of piprot I keep getting an error on my requirements file. I get the same error running it on the remot requirements for piprot as per the readme example.

```
$ piprot -g sesh/piprot
Traceback (most recent call last):
  File "/usr/local/bin/piprot", line 9, in <module>
    load_entry_point('piprot==0.9.0', 'console_scripts', 'piprot')()
  File "/usr/local/lib/python2.7/site-packages/piprot/piprot.py", line 386, in piprot
    url=cli_args.url)
  File "/usr/local/lib/python2.7/site-packages/piprot/piprot.py", line 216, in main
    response=result['latest']\
  File "/usr/local/lib/python2.7/site-packages/piprot/piprot.py", line 125, in get_version_and_release_date
    versions = {parse_version(v): v for v in response['releases'].keys() if not parse_version(v).is_prerelease}
  File "/usr/local/lib/python2.7/site-packages/piprot/piprot.py", line 125, in <dictcomp>
    versions = {parse_version(v): v for v in response['releases'].keys() if not parse_version(v).is_prerelease}
AttributeError: 'tuple' object has no attribute 'is_prerelease'
```

Reverting to 0.8.2 works properly on my requirements file. I don't understand what is wrong :(

**Comment by bennullgraham** [comment]
Looks like this was fixed in c93317f0, any chance of getting that code onto pypi Brenton? :)

**Comment by sesh** [comment]
@arnaudlimbourg @bennullgraham just pushed `0.9.1` to PyPI that includes the fix.

Sorry about that!
